iso690-full-note-cs.csl

```diff
@@ -233,17 +233,15 @@
   <macro name="issued">
     <choose>
       <if type="article-journal article-magazine article-newspaper" match="any">
-        <group delimiter=", ">
-          <date variable="issued">
-            <date-part name="year" range-delimiter="&#8211;"/>
-          </date>
-          <choose>
-            <if variable="volume">
-              <text term="volume" form="short" suffix="&#160;"/>
-              <text variable="volume"/>
-            </if>
-          </choose>
-        </group>
+        <date variable="issued">
+          <date-part name="year" range-delimiter="&#160;&#8211;&#160;"/>
+        </date>
+        <choose>
+          <if variable="volume">
+            <text prefix=", " term="volume" form="short" suffix="&#160;"/>
+            <text variable="volume"/>
+          </if>
+        </choose>
```

Comment:
@adam3smith, this would be better, right?

```suggestion
        <group delimiter=", ">
          <date variable="issued">
            <date-part name="year" range-delimiter="&#160;&#8211;&#160;"/>
          </date>
          <group delimiter="&#160;">
            <text term="volume" form="short"/>
            <text variable="volume"/>
          </group>
        </group>
```